src/CommaSeparatedIntParser.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+class CommaSeparatedIntParser extends AbstractValueParser
+{
+    protected function parse($value)
+    {
+        if (empty($value)) {
+            return null;
+        }
+        $value = explode(',', $value);
+        return array_map('intval', $value);
```

Comment:
I agree, I wasn't really sure about the expected behaviour but I assume you are right, let's return null for non-integers